core/claims/claimBasic.go

```diff
@@ -1,85 +1,70 @@
 package claims
 
 import (
-	"encoding/binary"
-
 	"github.com/iden3/go-iden3-core/merkletree"
 )
 
 const (
-	// 56+248+248+248=800 bits
-	indexSlotBits  = 800
+	// 152+248+248+248=896 bits
+	indexSlotBits  = 896
 	IndexSlotBytes = indexSlotBits / 8
 	// 216+248+248+248=960 bits
-	dataSlotBits  = 960
-	DataSlotBytes = dataSlotBits / 8
+	valueSlotBits  = 960
```

Comment:
this shouldn't be 152+248+248+248=896?

Basically we moved 64 bits from Index to Value, so Index was 800 bits and Value was 960, moving those 64 bits from I to V, results with 864 bits for Index, and 896 bits for Value, witch matches with the numbers calculated in the review comments.
See the Claims specification docs for more details

Reply:
I used 216 because the expiration date is not being used in this claim.

Reply:
Right, perfect